File: product/akbun-agent-analysiscode/workspace/fixtures/sample-msa/order-service/events.py
"""Event publishing to the shared message broker."""

import json
import os

import redis

broker = redis.Redis.from_url(os.environ.get("BROKER_URL", "redis://broker:6379"))


def publish(topic: str, payload: dict) -> None:
  """Publish one event to the broker."""
  broker.publish(topic, json.dumps(payload))

File: product/akbun-agent-k8s/workspace/fixtures/sample-msa/payment-service/events.js
// Event publishing to the shared message broker.
const { createClient } = require('redis');

const broker = createClient({ url: process.env.BROKER_URL || 'redis://broker:6379' });

function publish(topic, payload) {
  broker.publish(topic, JSON.stringify(payload));
}

module.exports = { publish };
